scripts/lib/acquisition-private-output.mjs

```
@@ -89,40 +130,75 @@ export function assertAcquisitionPrivatePathParents(
+    let closeError;
+    try {
+      fileSystem.closeSync(descriptor);
+    } catch (error) {
+      closeFailed = true;
+      closeError = error;
+    }
+    if (!accepted || closeFailed) {
+      cleanupIdentityMatchedPath(path, createdMetadata, fileSystem);
+    }
+    if (closeFailed && !operationFailed) {
+      throw closeError;
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)